File: 782-JewelsAndStones/782-JewelsAndStones.py
# Last updated: 6/25/2026, 9:12:49 AM
class Solution(object):
    def numJewelsInStones(self, jewels, stones):
        """
        :type jewels: str
        :type stones: str
        :rtype: int
        """
        d=Counter(stones)
        count=0
        for i in jewels:
            if i in d:
                count+=d[i]
        return count

File: 782-JewelsAndStones/782-JewelsAndStones.scala
// Last updated: 6/25/2026, 9:12:53 AM
object Solution {
    def numJewelsInStones(J: String, S: String): Int = {
        
    var result=0
        
    println (S.filter(J.contains(_)).length())
        
    for( x<- J ;y <-  S)
        {
            if (x==y)
            {
                result = result+1
            }
            
        }
    result
    }
}